week-5/F-webchat/exercise-1.js

```diff
@@ -33,5 +34,11 @@ When you open index.html in your browser, it should display the existing message
 
 */
 
+// Write your code here
```

Comment:
Exercise-3.js is missing

Reply:
it's in exercise-1 ,,,,, coz it says we have to add the setInterval to exercise-1